README.md

```diff
@@ -64,10 +64,12 @@ you'll need to do the following:
   - Presubmit jobs run on unmerged code in PRs
   - Postsubmit jobs run after merging code
   - Periodic job run on a timed basis
-* (Deprecated!) Some old jobs still run on jenkins
-  - Please do not add new jobs to jenkins
-  - Jenkins configuration is defined at `jenkins/job-configs`
-  - More deprecated details at [jenkins/README.md](jenkins/README.md)
+
+The configs need to be sorted, or else presubmit will fail.
+To keep them sorted, you can run:
+```
+bazel run //jobs:config_sort
+```
```

Comment:
maybe just mention `hack/update-config.sh`? /shrug

Reply:
They need to be sorted *and* kubernetes must be in sync with the security repo

Reply:
done